platform/mellanox/cmis_host_mgmt/cmis_host_mgmt.py

```diff
@@ -0,0 +1,151 @@
+
+        for field in summary:
+            key, value = field.split(": ")
+            
+            if key == 'Platform':
+                platform = value
+
+            elif key == 'HwSKU':
+                sku = value
+
+        return platform, sku
+
+
+    @staticmethod
+    def remove_im_file(file_path):
+        if os.path.isfile(file_path):
+            os.remove(file_path)
+
+
+    @staticmethod
+    def copy_im_file(src_path, dest_path):
+        if os.path.isfile(src_path):
+            shutil.copy(src_path, dest_path)
+
+
+    @staticmethod
+    def disable_im():
+        platform, sku = CMISHostMgmtActivator.parse_show_platform_summary()
+        sku_path = '/usr/share/sonic/device/{0}/{1}'.format(platform, sku)
+        platform_path = '/usr/share/sonic/device/{0}'.format(platform)
+        CMISHostMgmtActivator.change_param("sai_profile", sku_path, 'disable')
+        CMISHostMgmtActivator.change_param("pmon_daemon_control", platform_path, 'disable')
+        
+        CMISHostMgmtActivator.remove_im_file('{0}/{1}'.format(sku_path, 'media_settings.json'))
+        CMISHostMgmtActivator.remove_im_file('{0}/{1}'.format(sku_path,'optics_si_settings.json'))
+        CMISHostMgmtActivator.remove_im_file('{0}/{1}'.format(platform_path, 'media_settings.json'))
+        CMISHostMgmtActivator.remove_im_file('{0}/{1}'.format(platform_path, 'optics_si_settings.json'))
+        CMISHostMgmtActivator.remove_im_file('{0}/{1}'.format(sku_path, 'pmon_daemon_control.json'))
+
+
+    @staticmethod
+    def enable_im(args):
+        platform, sku = CMISHostMgmtActivator.parse_show_platform_summary()
+        sku_path = '/usr/share/sonic/device/{0}/{1}'.format(platform, sku)
+        platform_path = '/usr/share/sonic/device/{0}'.format(platform)
+
+        sku_num = re.search('[0-9]{4}', sku).group()
+        CMISHostMgmtActivator.PARAMS["sai_xml"]["file_name"] = "sai_{0}.xml".format(sku_num)
+
+        CMISHostMgmtActivator.copy_im_file(args[0], sku_path)
+        CMISHostMgmtActivator.copy_im_file(args[1], sku_path)
+        CMISHostMgmtActivator.copy_im_file('{0}/{1}'.format(platform_path, 'pmon_daemon_control.json'), sku_path)
+
+        CMISHostMgmtActivator.change_param("sai_profile", sku_path, 'enable')
+        CMISHostMgmtActivator.change_param("pmon_daemon_control", sku_path, 'enable')
+        CMISHostMgmtActivator.change_param("sai_xml", sku_path, 'enable')
+        
+
+@click.command()
+@click.option('--disable', is_flag=True, help='Disable CMIS Host Management')
+@click.option('--enable', nargs=2, type=click.Path(), help='Enable CMIS Host Management, receives two arguments: media_settings.json path, and optics_si_settings.json path')
+def main(disable, enable):
+
+    import pdb; pdb.set_trace()
+    if disable and enable:
+        print("Error: can't use both options, please choose one.")
+
+    if disable:
+        CMISHostMgmtActivator.disable_im()
+
+    elif enable:
+        CMISHostMgmtActivator.enable_im(enable)
+
```

Comment:
If user didn't provide disable/enable flag, maybe it's better to print to the screen that nothing was executed. 
Do we have "force" to one of the parameters in "click"?

Reply:
Fixed